crates/goose/src/hints/load_hints.rs

```diff
@@ -350,6 +341,57 @@ mod tests {
         assert!(hints.contains("Subdir .goosehints content"));
     }
 
+    #[test]
+    #[serial]
```

Comment:
serial doesn't protect you; it just means that this test will run serial with all the other serial tests. we're changing the current directory here, so that will make other tests flaky. do we need to change the current directory?